src/services/AudiusBackend.js

```diff
@@ -936,8 +937,31 @@ class AudiusBackend {
     }
   }
 
+  static async fetchUserAssociatedWallets(user) {
```

Comment:
do you want token dashboard sagas to re-use this function too?

Reply:
yeah can do